test/spec/controllers/inviteToGroupCtrlSpec.js

```diff
@@ -93,6 +97,10 @@ describe('Invite to Group Controller', function() {
     });
 
     context('email', function() {
+      beforeEach(function () {
+        sinon.stub(rootScope, 'hardRedirect');
```

Comment:
I guess it doesn't matter that much since rootScope gets overwritten on every test.